contrib/ci/sssd.supp

```diff
@@ -256,3 +257,14 @@
    fun:_dlerror_run
    fun:dlopen@@GLIBC_2.34
 }
+
```

Comment:
The leak is triggered when calling [DEBUG_CLI_INIT()](https://github.com/SSSD/sssd/pull/6327/files#diff-66b8dafcfa0268e48d1710e0527a63cb3d0cc0bffe79dbcf0d7865bea24a85c7R627) from the unit test environment and this wouldn't happen in a real production environment.

Reply:
I guess my question is why do we need this for the cmocka fido2 tests, and not other tests calling DEBUG_CLI_INIT ?

Reply:
Because I'm the first person to test the argument parsing in the unit-tests. This causes the debug initialization to be initialized two times. The first one in the unit-test itself: DEBUG_CLI_INIT() -> DEBUG_INIT(). And the second one in the production code: parse_arguments() -> DEBUG_INIT(). Apparently valgrind doesn't like this.

Reply:
Thanks for the explanation, in that case could the suppression be changed to specify these function names (test_parse_required_args, test_parse_all_args)? IMO that will ensure the suppressions are tied to these tests specifically

Reply:
I had to create an 'unnecessary' intermediate function, `test_parse_supp_valgrind_args()`, to tie everything correctly but it works. Thanks for the feedback!